readers/cassandra/messages.go

```diff
@@ -48,6 +48,17 @@ func (cr cassandraRepository) ReadAll(chanID string, rpm readers.PageMetadata) (
 		ALLOW FILTERING`, q)
 	countCQL := fmt.Sprintf(`SELECT COUNT(*) FROM %s WHERE channel = ? %s ALLOW FILTERING`, format, q)
 
+	fmt.Println()
```

Comment:
What are these calls to `fmt.Println()`? An attempt to add `\n`? Whole this printing looks strange...